osi_trafficcommandupdate.proto

```diff
@@ -0,0 +1,74 @@
+syntax = "proto2";
+
+option optimize_for = SPEED;
+
+import "osi_version.proto";
+import "osi_common.proto";
+
+package osi3;
+
+//
+// \brief This message enables the traffic participant model to send updates 
+// to the scenario engine about the execution of its received \c TrafficCommand input. 
+// While traffic actions are usually executed successfully by the traffic participant 
+// there may be actions which the traffic participant is not able to execute
+// either for capability or situation-specific reasons. This message gives
+// the traffic participant the basic possiblity to send feedback if an action 
+// cannot happen as requested by the \c TrafficCommand. Currently, it is out of
+// scope to standardize the exact reason for non-executability or failed execution 
+// because the reason can have multiple explantions. The point in time 
+// for this message to be sent is only restricted to be after (or at the same time) 
+// the \c TrafficCommand with the corresponding traffic action(s) has been sent. The 
+// responsibility for deciding about successful or unsuccessful scenario execution
+// lies fully on the side of the scenario engine.  
+//
+// \note This interface is currently just a placeholder and could be
```

Comment:
How should a user of OSI interpret this?

Is it meant to suggest that this message is more likely than others to be changed in a breaking way in a future major release?

Reply:
To your second question: I think, yes. A main motivation for TrafficCommand and TrafficCommandUpdate was to be well compatible to OpenSCENARIO semantically which is changing a lot, currently (OpenSCENARIO 1.x and 2.x, but also each stream itself). I think this is why we have a similar comment in https://github.com/OpenSimulationInterface/open-simulation-interface/blob/master/osi_trafficcommand.proto lines 15-18. This should probably be discussed more broadly in a meeting with OpenSCENARIO liaisons and fixed in a major OSI version.

Reply:
OK, thanks.  Assuming this isn't intending to imply that the interface may change in a breaking way as part of a minor release I'm happy to leave as-is.

(if we do want that flexibility, then I think it is worth calling that out very explicitly)